server/server.d.ts

```diff
@@ -0,0 +1,7 @@
+import { Request } from 'koa'; 
+
+declare module 'koa' { 
+  interface Request { 
+    body: any;
```

Comment:
=)

Reply:
The 'body' really has any type. It's data that a user sends to the server. We can't setup types((